site/src/sphinx/server-grpc.rst

```diff
@@ -110,7 +116,33 @@ to just pass ``GrpcSerializationFormats.values()``.
 
 The server will support both native gRPC_ and gRPC-Web_ from the same endpoint. Use the unofficial
 gRPC-Web-Client_ to access the service from a browser. gRPC-Web_ does not support RPC methods with streaming
-requests.
+requests. gRPC-Web-Client_ might first send ``preflight`` requests by the HTTP ``OPTIONS`` method, in order
```

Comment:
Can we add `if the origin of the Javascript and API server are different`? I don't want it to seem like `CorsService` should always be applied.

Reply:
Added! Thanks for the good point that I missed :)